app/Http/Controllers/FoodController.php

```diff
@@ -3,8 +3,63 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Food;
+use App\Category;
+use App\Http\Requests\FoodPostRequest;
+use Image;
 
 class FoodController extends Controller
 {
-    //
+    protected $food;
+    
+    /**
+     * FoodController constructor.
+     *
+     * @param Food $food dependence injection
+     */
+    public function __construct(Food $food)
+    {
+        $this->food = $food;
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $categories = Category::orderBy('id', 'DESC')->get();
+        return view('foods.create', ['categories' => $categories]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request of food
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(FoodPostRequest $request)
+    {
+        $arrFoods = $request->all();
+        $arrFoods = $request->except(['_token']);
+        if ($request->hasFile('image')) {
+            $file = $request->file('image');
+            $fileName = time() . "-" . $file->getClientOriginalName();
+            $arrFoods['image'] = $fileName;
+        } else {
+            $arrFoods['image'] = 'default.jpg';
```

Comment:
You already created `config/constant.php`. Please put `default.jpg` to `constant.php`